DigitalLearningSolutions.Web/Views/MyAccount/EditDetails.cshtml

```diff
@@ -8,18 +8,17 @@
 @{
   var errorHasOccurred = !ViewData.ModelState.IsValid;
   ViewData["Title"] = errorHasOccurred ? "Error: Edit Details" : "Edit Details";
+  var application = Model.Application ?? ApplicationType.Main;
+  ViewData["Application"] = application.ApplicationName;
+  ViewData["HeaderPath"] = application.HeaderPath;
+  ViewData["HeaderPathName"] = application.HeaderPathName;
+  var aspAllRouteData = new Dictionary<string, string?>() {

```

Comment:
I'd call this something like `routeParamsForIndexLink`. That's a bit more specific and easier to understand than `aspAllRouteData`.
Similar in other places.